addon/templates/components/frost-error.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="frost-error-description" data-test={{hook (concat hook '-description')}}>
```

Comment:
`Component`s extending from the `ember-frost-core` component have a nice property named `css` that automagically get the name of the component so you can write the class as:

`<div class='{{css}}-description'...`

Which protects you if the component is renamed in the future (plus it's shorter)

If you do the same for a child component the format is
```
{{some-component
  class=(concat css '-something')
```